api/accounts.js

```diff
@@ -1,20 +1,21 @@
 const accounts = require('../lib/api/accounts');
+const handler = require('./handler');
 
 module.exports = function (app) {
-	const api = new accounts(app);
+	const accountApi = new accounts(app);
 
-	app.get('/api/getAccount', (req, res, next) => {
-		api.getAccount(
-			req.query,
-			(data) => { res.json(data); },
-			(data) => { req.json = data; return next(); });
-	});
+	app.get('/api/getAccount', (req, res, next) =>
+		handler(accountApi, 'getAccount', req.query, req, res, next));
 
-	app.get('/api/getTopAccounts', (req, res) => {
-		api.getTopAccounts(
-			{ offset: req.query.offset,
-				limit: req.query.limit },
-			(data) => { res.json(data); },
-			(data) => { res.json(data); });
-	});
+	app.get('/api/getTopAccounts', (req, res, next) =>
```

Comment:
You don't need to write duplicated handlers for each route with the same path algorithm. you can define just one route handler which uses a default global handler. and in this global handler, you can customize your request data and call your handler.
take a look at this sample:
```javascript
const serviceHandler = (req, res, next) => {
	let serviceName = req.params.service
	let serviceData = {}
	switch(serviceName) {
		case 'getTopAccounts':
			serviceData = {
				offset: req.query.offset,
            	                limit: req.query.limit
			}
		break
	}
	handler(accountApi, serviceName, serviceData, req, res, next)
}

app.get('/api/:service', serviceHandler)
```

Reply:
take a look at next review. I wrote your codes in `(/api/blocks.js)` in a different way